pkg/resources/management.cattle.io/v3/setting/Setting.md

```diff
@@ -9,6 +9,7 @@ When settings are created or updated, the following common checks take place:
 - If set, `user-last-login-default` must be a date time according to RFC3339 (e.g. `2023-11-29T00:00:00Z`).
 - If set, `user-retention-cron` must be a valid standard cron expression (e.g. `0 0 * * 0`).
 - The `auth-user-session-ttl-minutes` must be a positive integer and can't be greater than `disable-inactive-user-after` or `delete-inactive-user-after` if those values are set.
+- The `auth-user-session-idle-ttl-minutes` must be a positive integer and can't be greater than `auth-user-session-ttl-minutes` and has to be >= 1 minute.
```

Comment:
smallest of all possible nits, but "positive integer" and "has to be >= 1" are redundant

Reply:
you're right, thanks